ircd/channel.c

```diff
@@ -2534,6 +2534,13 @@ int	m_join(aClient *cptr, aClient *sptr, int parc, char *parv[])
 
 		if (!chptr)
 		{
+			if(IsSplit() && UseModes(name) && *name != '&')
```

Comment:
`&& !(*name == '&' || *name == '!')`
Since !channels are collision proof nothing should prevent their creation during split-mode

Reply:
Thank you, I will change it.